wcomponents-core/src/test/java/com/github/bordertech/wcomponents/container/TransformXMLInterceptor_Test.java

```diff
@@ -59,26 +59,12 @@ public static void tearDownClass() {
 	}
 
 	/**
```

Comment:
Not really happy with this. It is OK to prove the change has been made but not really useful after the initial pull.